docs/UserGuide.md

```diff
@@ -52,15 +51,15 @@ Amigos is a desktop application to help tech-savvy university students manage th
 * If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
   e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
-  e.g. if the command specifies `help 123`, it will be interpreted as `help`.
+* Extraneous parameters for commands that do not take in parameters (such as `showfriends`, `showevents`, `exit` and `clear`) will be ignored.<br>
+  e.g. if the command specifies `showfriends 123`, it will be interpreted as `showfriends`.
 
 </div>
 
 
 ### Adding a friend: `addfriend`
```

Comment:
Should we keep the tags feature from AB3? It looks like we included it in the mockup.